client/src/themes/theme.js

```diff
@@ -9,6 +18,6 @@ export const theme = createMuiTheme({
     },
   },
   palette: {
-    primary: { main: "#0d79de" },
+    primary: { main: "#3C8FFF", gray: "#F5F7FB" },
```

Comment:
:+1: 
Now that we have this here, go ahead and use this the 4 other places you use this color (`backgroundColor: theme.palette.primary.gray`, for example).